source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
Add Video Content or link if Possible

Reply:
If it would have possible then it would have added.